### PR TITLE
Fix nested object serialization with generics

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

Enums implementation caused an issue with serializing nested objects when using the generic versions of methods in Swift/Apple

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -262,8 +262,8 @@ docker run --rm -v $(pwd):$(pwd):rw -w $(pwd) -v /var/run/docker.sock:/var/run/d
   * **description** -> Description of Appwrite SDK
   * **namespace** -> SDK Namespace
   * **version** -> SDK Version
-  * **endpoint** -> Default Endpoint (example: "https://cloud.appwrite.io/v1")
-  * **host** -> Default Host (example: "cloud.appwrite.io")
+  * **endpoint** -> Default Endpoint (example: "https://appwrite.io/v1")
```
Comment (lohanidamodar) [inline CONTRIBUTING.md:265]: Why did this change?

File: src/SDK/Language.php [hunk 2]
```diff
@@ -108,7 +108,6 @@ protected function toCamelCase($str): string
 
         $str = \preg_replace('/[^a-zA-Z0-9]+/', ' ', $str);
         $str = \trim($str);
-        $str = strtolower($str);
```
Comment (lohanidamodar) [inline src/SDK/Language.php:111]: how is this relevant for nested serialization?

**Review by lohanidamodar — COMMENTED** [review]
rest looks good

File: src/SDK/Language.php (hunk 2 above)
Comment (abnegate) [reply src/SDK/Language.php]: This was stacked from the other PR

File: CONTRIBUTING.md (hunk 1 above)
Comment (abnegate) [reply CONTRIBUTING.md]: Must have been initially fixed in the reverted PR, fixed here

**Review by lohanidamodar — APPROVED** [review]
